rest/src/main/java/org/pdxfinder/controllers/AjaxController.java

```diff
@@ -252,90 +236,6 @@ public Object exportMappingData(HttpServletResponse response,
     return csvReport;
   }
 
-
-  /****************************************************************
-   *                   INTERACTIONS WITH ZOOMA                    *
-   ****************************************************************/
```

Comment:
Was this the duplication you had mentioned?

Reply:
That is code for endpoints that are not being used in the UI, and they didn't work well either, so they are obsolete.